### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by akosveres** [eggheads/eggdrop-docker#5]
README.md sais:

> An easy way to add scripts would be to create a scripts directory on the host and mount it to /home/eggdrop/eggdrop/data. This would be accomplished by adding an option similar to
> 
> -v /path/to/host/scripts:/home/eggdrop/eggdrop/scripts
> to your docker run command line (and of course, don't forget to edit your configuration file to actually load it!)
> 

Is it /home/eggdrop/eggdrop/data or /home/eggdrop/eggdrop/scripts ?

**Comment by vanosg** [comment]
I started this response on a very different train of thought, and wrote why it should be mounted to one place over another- but then I realized you were just pointing out my inconsistency in documentation paths! 

You could do it wherever you want, as long as your eggdrop config points to the same place. But, your question did get me thinking and realize that mounting a host dir to eggdrop/scripts would replace the docker.tcl script that is currently sourced from there (thus causing a crash on start), so I'll push something to correct that use case. 

Bottom line though, just match your -v mount to whatever you add in the config file- it could be /home/eggdrop/eggdrop/banana/hammock/partytime, as long as the path in your config matches! 

Thanks for the input! I'll update the documentation, and also fix this use case!

**Comment by akosveres** [comment]
Yes, that was my thought as well, since /scripts is used by other scripts, especially with more tcl files, it would replace everything. Thx for the feedback.

**Comment by vanosg** [comment]
Added an error check to the config, so that if docker.tcl isn't found, eggdrop can still continue its boot. I figure this is ok since all docker.tcl does is yell at the user if they haven't mounted a dir for persistence. If the user is mounting a scripts dir, I think it's fair to assume they understood what docker.tcl is telling them to do, so no loss to not have that nag occur. 

Thanks again
